doc/source/user_guide/advanced.rst

```diff
@@ -177,6 +177,37 @@ completely analogous way to selecting a column in a regular DataFrame:
 See :ref:`Cross-section with hierarchical index <advanced.xs>` for how to select
 on a deeper level.
 
+.. _advanced.set_levels:
```

Comment:
I would rather just edit the actual set_levels doc-string with examples. This is too much for the main docs (and not generally what someone would want).